src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -539,7 +595,7 @@ export class SearchSource {
       if (!body.hasOwnProperty('_source')) {
         body._source = sourceFilters;
       }
-      if (body._source.excludes) {
+      if (body._source) {
```

Comment:
This will always evaluate to `true` and can probably be removed. (If `body._source` is undefined, we assign it on L596 above, and `index.getSourceFiltering()` will always be truthy).